### PR TITLE
Add the feature of searching the person on Google Map

### PR DESCRIPTION
**Pull request by YuchenHe98**

### CODE REVIEW

File: src/main/java/seedu/address/MainApp.java [hunk 1]
```diff
@@ -20,21 +14,17 @@
 import seedu.address.commons.util.StringUtil;
 import seedu.address.logic.Logic;
 import seedu.address.logic.LogicManager;
-import seedu.address.model.AddressBook;
-import seedu.address.model.Model;
-import seedu.address.model.ModelManager;
-import seedu.address.model.ReadOnlyAddressBook;
-import seedu.address.model.UserPrefs;
+import seedu.address.model.*;
```
Comment (ghost) [inline src/main/java/seedu/address/MainApp.java]: Avoid * imports

File: src/main/java/seedu/address/commons/events/ui/LocateCommandEvent.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    @Override
+    public String toString() {
+        return this.getClass().getSimpleName();
+    }
+
+    public ReadOnlyPerson getPerson() {
+        return person;
+    }
+}
```
Comment (ghost) [inline src/main/java/seedu/address/commons/events/ui/LocateCommandEvent.java]: Need a new line after EOF

File: src/main/java/seedu/address/commons/events/ui/LocateCommandEvent.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
+package seedu.address.commons.events.ui;
+
+import seedu.address.model.person.Person;
+import seedu.address.commons.events.BaseEvent;
+import seedu.address.model.person.ReadOnlyPerson;
+
+
+public class LocateCommandEvent extends BaseEvent{
```
Comment (ghost) [inline src/main/java/seedu/address/commons/events/ui/LocateCommandEvent.java]: Need a spacing preceding {

File: src/main/java/seedu/address/logic/commands/LocateCommand.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
+package seedu.address.logic.commands;
+
+import seedu.address.commons.core.EventsCenter;
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.events.ui.LocateCommandEvent;
+import seedu.address.commons.events.ui.ShowHelpRequestEvent;
```
Comment (ghost) [inline src/main/java/seedu/address/logic/commands/LocateCommand.java]: Delete unnecessary import statements

File: src/main/java/seedu/address/logic/commands/LocateCommand.java [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.person.ReadOnlyPerson;
+import seedu.address.model.person.exceptions.PersonNotFoundException;
+
+import java.util.List;
```
Comment (ghost) [inline src/main/java/seedu/address/logic/commands/LocateCommand.java]: Check order of import statements

File: src/main/java/seedu/address/logic/commands/LocateCommand.java [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    public static final String MESSAGE_LOCATE_PERSON_SUCCESS = "Showing the address of person ";
+
+    private final Index targetIndex;
+
+    public LocateCommand(Index targetIndex) {
+        this.targetIndex = targetIndex;
+    }
+
+    @Override
+    public CommandResult execute() throws CommandException{
```
Comment (ghost) [inline src/main/java/seedu/address/logic/commands/LocateCommand.java]: Missing another white space preceding { 
Reasons why Travis is failing

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 7]
```diff
@@ -6,19 +6,7 @@
...
-import seedu.address.logic.commands.EditCommand;
-import seedu.address.logic.commands.ExitCommand;
-import seedu.address.logic.commands.FindCommand;
-import seedu.address.logic.commands.HelpCommand;
-import seedu.address.logic.commands.HistoryCommand;
-import seedu.address.logic.commands.ListCommand;
-import seedu.address.logic.commands.RedoCommand;
-import seedu.address.logic.commands.SelectCommand;
-import seedu.address.logic.commands.UndoCommand;
+import seedu.address.logic.commands.*;
```
Comment (ghost) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java]: Avoid * imports

File: src/main/java/seedu/address/model/person/Person.java [hunk 8]
```diff
@@ -146,9 +146,7 @@ public String toString() {
     public void removeTag(String str) {
         Set<Tag> tagset = getTags();
         for (Tag t : tagset) {
-            if (t.isSame(str)) {
-                tagset.remove(t);
-            }
+            if (t.isSame(str)) tagset.remove(t);
```
Comment (ghost) [inline src/main/java/seedu/address/model/person/Person.java]: Travis : Put 'if' statement in braces { }

**Comment by YuchenHe98** [comment]
fixing on it...

**Pull request merged**